.lando.yml

```diff
@@ -20,7 +20,7 @@ services:
   cache:
     type: 'redis:6'
   node:
-    type: 'node:22'
+    type: 'node:24'
```

Comment:
Checked supported versions on this page:
https://docs.lando.dev/plugins/node/

![Image](https://github.com/user-attachments/assets/05990682-29b7-41fa-bedf-50af2b7caacc)